A6g Inconsistency Fix (#1121)
thewca/wca-regulations: wca-regulations.md

        - A6e2) Penalty for touching the puzzle after stopping the timer without applying any moves: time penalty (+2 seconds). Exception: if immediately after stopping the timer, the competitor briefly touches the puzzle without affecting its state, the time penalty may be lifted, at the discretion of the judge (see [Regulation 11f](regulations:regulation:11f)).
    - A6f) The competitor must not reset the timer until both the competitor and the judge have signed the score sheet. Penalty: disqualification of the attempt (DNF), at the discretion of the judge.
        - A6f1) If the competitor resets the timer before the result has been completely recorded, the judge must not write down the result from memory or from video or photographic evidence, and must disqualify the attempt instead (DNF).

>     - A6g) The judge determines whether the puzzle is solved. They must not touch the puzzle before they have determined the state of the puzzle. Exception: For Clock, the judge will usually need to pick up the puzzle to verify both faces.

Reviewer: ```suggestion
    - A6g) The judge determines whether the puzzle is solved. If the puzzle is misaligned, it must not be touched before the judge has determined its final state.
```

As far as I know, this rule is supposed to prevent anyone from touching the puzzle when it has a misalignment. So I think something like this should work.
